gnuradio-runtime/lib/basic_block.cc

```diff
@@ -35,6 +35,7 @@ basic_block::basic_block(const std::string& name,
       d_unique_id(s_next_id++),
       d_symbolic_id(global_block_registry.block_register(this)),
       d_symbol_name(global_block_registry.register_symbolic_name(this)),
+      d_registry_name(name + std::to_string(d_symbolic_id)),
```

Comment:
Would it be a solution to use `d_symbolic_id` directly?
There's also `d_unique_id` which might be another alternative.

Reply:
`d_symbolic_id` is not unique, and `d_unique_id` is not the correct type but it looks like we can use `d_symbol_name`. This significantly simplifies this PR so I'll get it updated as soon as i am able.

Reply:
`d_symbol_name` is the default value of `alias()`. This seems like the right way to go.